src/libraries/System.Runtime.InteropServices/System.Runtime.InteropServices.sln

```diff
@@ -7,12 +7,24 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Private.CoreLib", ".
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestUtilities", "..\Common\tests\TestUtilities\TestUtilities.csproj", "{1FF4CC8E-49C3-42A0-A6E0-2E5908455FBA}"
```

Comment:
Are we happy with the SLN updates here?

Reply:
Yes, they're needed as we're now referencing System.Collections.Concurrent from System.Runtime.InteropServices.